llama-index-cli/pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.mypy]
+disallow_untyped_defs = true
+# Remove venv skip when integrated with pre-commit
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+authors = ["llamaindex"]
+description = "llama-index cli"
+license = "MIT"
+maintainers = [
+    "Andrei Fajardo <andrei@runllama.ai>",
+    "Haotian Zhang <ht@runllama.ai>",
+    "Jerry Liu <jerry@llamaindex.ai>",
+    "Logan Markewich <logan@llamaindex.ai>",
+    "Simon Suo <simon@llamaindex.ai>",
+    "Sourabh Desai <sourabh@llamaindex.ai>",
+]
+name = "llama-index-cli"
+packages = [{include = "llama_index/"}]
+readme = "README.md"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<3.12"
```

Comment:
btw, I've noticed we have `<3.12` in all our packages. We might want to consider making this `<4.0.0` 🤔

Reply:
![image](https://github.com/run-llama/llama_index/assets/92402603/4beb201b-50b3-4a6e-9b1c-5bf2198a8f1e)
Shoot, so in order to get back "python = 3.8.1<4.0" i think we have to update all pyproject.toml and re-build.

Reply:
It can be done ofc, but it might be a bit more involved then just changing pyproject.toml's :)